android/assets/jsons/translations/Simplified_Chinese.properties

```
@@ -1455,23 +1455,20 @@ Enhanced religion = 强化宗教
 # feel free to not translate these strings for now
 
 Spy = 间谍
- # Requires translation!
-Spy Hideout = 
- # Requires translation!
-Spy present = 
- # Requires translation!
-Move = 
+Spy Hideout = 间谍正藏身于
```

Not sure weather this is right. Hideout means a place to hide in English. Can probably leave it blank until the spy release.
```suggestion
Spy Hideout = 
```
or:
```suggestion
Spy Hideout = 间谍隐匿处
```